modules/user-access/pages/access-control-model.adoc

```diff
@@ -20,25 +24,130 @@ This allows the user to run commands such as `ls` and `SHOW VERTEX` on the graph
 
 To view a complete list of privileges available in TigerGraph and the commands they enable a user to run, see xref:reference:list-of-privileges.adoc[List of Privileges].
 
-=== Global vs local privileges
+=== Privilege scopes
+
+There are four different scopes for a privilege:
+
+* Global
+* Graph-level
+* Type-level (Only applicable to <<_data_crud_privileges>>)
+* Attribute-level (Only applicable to <<_data_crud_privileges>>)
+
+Global privileges apply to all graphs and global objects.
+Graph-level privileges only apply on the graph they belong to.
+Type-level privileges only apply to specific vertex or edge types on a graph.
+Attribute-level privileges only apply to specific attributes of a vertex or edge type.
+
+A privilege on a larger scope includes the privilege on a lower scope encompassed by the larger scope.
+If you have a privilege on the graph level, you also have that privilege on the type level and attribute level for all types and attributes in that graph.
+If you have a privilege on the global scope, you have that privilege across all graphs, types and attributes.
+
+Most privileges are global or graph-level.
+Only privileges pertaining to graph data (`CREATE_DATA`, `READ_DATA`, `UPDATE_DATA`, `DELETE_DATA`) can be granted on a type level or attribute level.
+
 
-The scope of a privilege can be global or local. Global privileges apply to all graphs and global objects. Local privileges only apply on the graph they belong to.
+For example:
 
-For example, a role with `WRITE_QUERY` on graph `social` can only create queries on graph `social`, but not on other graphs. In contrast, a role with `WRITE_QUERY` on the global scope can create queries on all graphs.
+* A role with `WRITE_QUERY` on graph `Social` can only create queries on graph `social`, but not on other graphs.
+In contrast, a role with `WRITE_QUERY` on the global scope can create queries on all graphs.
+* A role with `UPDATE_DATA` on the `age` attribute of the `person` vertex on graph `social` are allowed to run queries that update the value of the `age` attribute of `person` vertices.
```

Comment:
* A role with `UPDATE_DATA` on the `age` attribute of the `person` vertex on graph `social` is allowed to run queries that update the value of the `age` attribute of `person` vertices.

Reply:
Also, make sure the variable names conform to the style guide (or at least are unified in this article)